dotnet/src/dotnetframework/GxClasses/Model/GXBaseObject.cs

```diff
@@ -164,7 +171,9 @@ private string Decrypt64(string value, string key, bool safeEncoding)
 			}
 			catch (InvalidKeyException)
 			{
+				context.SetCookie("GX_SESSION_ID", string.Empty, string.Empty, DateTime.MinValue, string.Empty, context.GetHttpSecure());
 				GXLogging.Error(log, "440 Invalid encryption key");
+				SendResponseStatus(440, "Session timeout");
```

Comment:
I got your point. But this method is not called when user uses Encrypt64 and Decrypt64 which are in [Encryption.cs](https://github.com/genexuslabs/DotNetClasses/blob/master/dotnet/src/dotnetcommon/GxEncrypt/Encryption.cs)
Those methods in GXBaseObject are used by genexus objects only. For .NET framework the equivalent code is at [GxHttp.cs](https://github.com/genexuslabs/DotNetClasses/blob/master/dotnet/src/dotnetframework/GxClasses/Middleware/GXHttp.cs), because GXHttpHandler does not have the GXBaseObject base class.

Reply:
Great!